components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/CmdUtils.java

```diff
@@ -442,6 +538,16 @@ public static String getMainConfigLocation() {
                 + CliConstants.MAIN_CONFIG_FILE_NAME;
     }
 
+    /**
+     * Returns location of the micro-gw.conf resource file
```

Comment:
Add `.` to the end of first/only sentence. If not there will be checkstyle issues

Reply:
Fixed. thanks. c048312